GoogleForCreators/web-stories-wp
# Media: Enable row-based gallery in Uploads tab

## Summary

Turns on feature where we use a row based gallery in the Uploads tab.

## User-facing changes

Gallery is now row based instead of column based.

## Testing Instructions

Verify that the usual operations work in this gallery arrangement as well. (eg. pagination, search, insert, edit, delete)

---



Fixes #3319
